project/settings.py

```diff
@@ -60,13 +60,13 @@
         CACHES = {
             'default': {
                 'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-                'LOCATION': '127.0.0.1:11211',
+                'LOCATION': os.environ.get('MEMCACHED_SERVERS', '127.0.0.1:11211'),
```

Comment:
```suggestion
                'LOCATION': os.environ.get('MEMCACHE_SERVERS', '127.0.0.1:11211'),
```

s/MEMCACHED/MEMCACHE/ - sorry to be a pain.

Reply:
Isn't the correct spelling "[memcached](https://memcached.org)" ?

Reply:
Seems to be an ongoing nightmare as to whether to call it memcache or memcache. However the existing convention we have with the env var is without the D.